lms/templates/admin/macros.html.jinja2

```diff
@@ -128,18 +130,22 @@
 {% endmacro %}
 
 
-{% macro instances_table(request, instances) %}
-{{ object_list_table(
-    request, 'admin.instance', instances,
-    fields=[
-        {"label": "Name", "name": "name"},
-        {"label": "Created", "name": "created"},
-        {"label": "Last Launched", "name": "last_launched"},
-        {"label": "Product Family", "name": "tool_consumer_info_product_family_code"},
-        {"label": "Email", "name": "requesters_email"},
-        {"label": "LMS URL", "name": "lms_url"},
-    ]
-) }}
+{% macro instances_table(request, instances, extra_fields=None) %}
+    {% if not extra_fields %}
```

Comment:
Nice trick with `extra_fields`

Reply:
This was one unintended advantage of the row based rather than column based approach.